install/kubernetes/istio-rbac-beta.yaml

```diff
@@ -77,6 +83,20 @@ roleRef:
   name: istio-ca
   apiGroup: rbac.authorization.k8s.io
 ---
+# Grant permissions to mixer.
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: istio-mixer-role-binding
+subjects:
+- kind: ServiceAccount
+  name: istio-mixer-service-account
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: istio-pilot
```

Comment:
this is really a big hack. @mandarjog or some mixer hacker to get the right permissions.